src/actions/formActions.js

```diff
@@ -0,0 +1,260 @@
+import { FORM_ACTIONS, PRODUCT_ACTIONS } from '../shared/actionConstants';
+import logout from '../apis/logoutApi';
+import { countryData } from '../apis/regionApi';
+
+export const logoutRequest = (token, dispatch) => {
```

Comment:
logout request function should not be a part of actions file, It should be mostly in container,

Reply:
removed from actions and used in container directly